src/Makefile

```
@@ -602,9 +607,15 @@ install_hashcat: $(HASHCAT_FRONTEND)
 	$(INSTALL) -m 755 -d                                                    $(DESTDIR)$(INSTALL_FOLDER)
 	$(INSTALL) -m 755 $(HASHCAT_FRONTEND)                                   $(DESTDIR)$(INSTALL_FOLDER)/
 
+.PHONY: install_tools
+install_tools:
+	$(INSTALL) -m 755 -d                                                    $(DESTDIR)$(INSTALL_FOLDER)
+	$(INSTALL) -m 755 ${TOOLS}                                              $(DESTDIR)$(INSTALL_FOLDER)/
+
 .PHONY: uninstall
 uninstall:
 	$(RM) -f  $(DESTDIR)$(INSTALL_FOLDER)/$(HASHCAT_FRONTEND)
+	cd $(DESTDIR)$(INSTALL_FOLDER) && $(RM) -f $(notdir ${TOOLS})
```

Why are you `cd`-ing instead of something like that?

```Makefile
.PHONY: uninstall
uninstall
  ...
  $(RM) -f  $(DESTDIR)$(INSTALL_FOLDER)/*hashcat.p[ly]
```

---

That would be the easy way, but: then we would be deleting everything that matches `*hashcat.p[ly]` in `/usr/local/bin/` or wherever. Which could be more than just the things we deployed there. So instead, I'm using the list of matching names from the `tools/` source dir. ...Which only works conveniently, if we are in `.../$(INSTALL_FOLDER)` at the time.

---

Hmm... right.

How about using `notdir` along with `addprefix`? I am really aware of using `cd` - it looks ugly.

```suggestion
	$(RM) -f  $(addprefix $(DESTDIR)$(INSTALL_FOLDER)/,$(notdir ${TOOLS}))
```

---

Great idea! Tested, and updated the PR.